app/components/channel_list/header/header.tsx

```diff
@@ -79,9 +79,7 @@ const ChannelListHeader = (props: Props) => {
                     />
                 </TouchableWithFeedback>
             </View>
-            <Text style={styles.subHeadingStyles}>
-                {props.subheading}
-            </Text>
+            <Text style={styles.subHeadingStyles}/>
```

Comment:
text value here is missing, I guess you want the value returned by `useServerDisplayName` right ?